api/v1/server/handlers/tenants/update.go

```diff
@@ -0,0 +1,39 @@
+package tenants
+
+import (
+	"github.com/labstack/echo/v4"
+
+	"github.com/hatchet-dev/hatchet/api/v1/server/oas/gen"
+	"github.com/hatchet-dev/hatchet/api/v1/server/oas/transformers"
+	"github.com/hatchet-dev/hatchet/internal/repository"
+	"github.com/hatchet-dev/hatchet/internal/repository/prisma/db"
+)
+
+func (t *TenantService) TenantUpdate(ctx echo.Context, request gen.TenantUpdateRequestObject) (gen.TenantUpdateResponseObject, error) {
+	tenant := ctx.Get("tenant").(*db.TenantModel)
```

Comment:
@abelanger5 do we need to do any further permission checks here? 

Do we want anyone in a tenant to be able to update or only Owners?

Reply:
Let's just allow anyone to update this for now, will do a more fine-grained permissions sweep once we have more to differentiate members from admins/owners.